helloextend-protection/admin/class-helloextend-protection-admin.php

```
@@ -160,10 +160,11 @@ public function enqueue_scripts()
         $ajaxurl           = admin_url('admin-ajax.php');
         $nonce             = wp_create_nonce('helloextend_sync_nonce');
         $helloextend_sync_batch = $this->helloextend_protection_for_woocommerce_settings_catalog_sync_options['helloextend_sync_batch'];
+        $debug_log_enabled = $this->helloextend_protection_for_woocommerce_settings_general_options['enable_helloextend_debug'];
```

PHPMD (2.15.0)

The variable $debug_log_enabled is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)